bedrock/firefox/tests/test_views.py

```diff
@@ -749,43 +749,49 @@ def test_yandex_scene_1_switch_off(self, render_mock):
         views.new(req)
         render_mock.assert_called_once_with(req, 'firefox/new/scene1.html', ANY)
 
-    # how to install experiment - issue 6704
+    # privacy dry market test - issue 6899
 
-    def test_install_scene_1_a(self, render_mock):
-        req = RequestFactory().get('/firefox/new/?v=a')
+    def test_privacy_dmt_scene_1_a(self, render_mock):
```

Comment:
Add `@patch.dict(os.environ, SWITCH_EXPERIMENT_FIREFOX_PRIVACY_DMT='True')` to each of these tests in order for them to pass.